src/main/resources/db/migration/V1__schema.sql

```diff
@@ -0,0 +1,30 @@
+-- GOAL
+CREATE TABLE IF NOT EXISTS goal
+(
+    id         BIGINT AUTO_INCREMENT,
```

Comment:
```suggestion
    id         BIGINT      AUTO_INCREMENT,
```
여기도 맞추는 건 어떻게 생각하시나요?

Reply:
좋습니당! 그런데 저 질문 있는데, java 코드는 인덴트가 2가 적용이 되는데, sql은 인덴트가 4더라구요!
이게 DduDuStyle.xml에 정의되어 있는 것일까요?

Reply:
그러게요 구글에서 받은거라 sql 인덴트는 따로 건든게 없긴한데 sql은 자바랑 별개로 보나봅니다!

Reply:
commit 6050b1d5982870ab22016a9ac401f055a9c88923 적용했습니다!